hooks/useClickOutside.ts

```diff
@@ -2,7 +2,7 @@ import { useEffect } from 'react';
 import type { RefObject, MouseEvent, TouchEvent } from 'react';
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
You should be able to remove this eslint warning now.

Reply:
Question is refer [here](https://github.com/nodejs/nodejs.org/pull/5349#discussion_r1183592983)